pkgs/by-name/zd/zdns/package.nix

```diff
@@ -6,16 +6,16 @@
 
 buildGoModule rec {
   pname = "zdns";
-  version = "2023-04-09-unstable";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "zmap";
     repo = pname;
-    rev = "ac6c7f30a7f5e11f87779f5275adeed117227cd6";
-    hash = "sha256-que2uzIH8GybU6Ekumg/MjgBHSmFCF+T7PWye+25kaY=";
+    rev = "v${version}";
```

Comment:
use tag

Reply:
You're right, will push an update in a second. I'm not sure why you removed `hash` though? Removing that leads to an error for me.